Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -887,7 +912,7 @@ void ACesium3DTileset::Tick(float DeltaTime) {
   }
 
   Cesium3DTiles::TilesetOptions& options = this->_pTileset->getOptions();
-  options.maximumScreenSpaceError = this->MaximumScreenSpaceError;
+  options.maximumScreenSpaceError = (double)this->MaximumScreenSpaceError;
```

Comment:
Is the explicit cast to double needed? If so, it'd be better to use a static_cast instead of a C-style cast.